R/utils.R

```diff
@@ -346,18 +346,17 @@ sanitize_terms <- function(x, standardized = FALSE) {
 }
 
 
-#' Prettify term names
+#' Prettify Term Names
```

Comment:
Do we really want to export this function at this point? Before exporting, I think we should, at the very least, reconsider the name of this function. I think 'prettify'  is not very descriptive.

Reply:
see above